homeassistant/components/tasmota/sensor.py

```diff
@@ -288,7 +289,7 @@ def sensor_state_updated(self, state: Any, **kwargs: Any) -> None:
             self._state = state
         self.async_write_ha_state()
 
-    @property
```

Comment:
It looks like these can happen in the constructor in a separate PR (need to verify)

Reply:
#100390